Files/Views/SettingsPages/Multitasking.xaml

```diff
@@ -76,6 +76,18 @@
                             TextWrapping="WrapWholeWords" />
                     </StackPanel>
                 </StackPanel>
+
+                <ToggleSwitch x:Uid="SettingsMultitaskingEnableDualPane"
+                              Header="Enable dual pane view"
+                              HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                              IsOn="{x:Bind AppSettings.IsDualPaneEnabled, Mode=TwoWay}" />
+
+                <ToggleSwitch x:Name="SettingsMultitaskingAlwaysOpenDualPane"
+                              x:Uid="SettingsMultitaskingAlwaysOpenDualPane"
+                              Header="Always open new tabs in dual pane mode"
+                              HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                              x:Load="{x:Bind AppSettings.IsDualPaneEnabled, Mode=OneWay}"
+                              IsOn="{x:Bind AppSettings.AlwaysOpenDualPaneInNewTab, Mode=TwoWay}" />
```

Comment:
Instead of reworking the open in new tab menu item, should there be a new menu item to open items in a new pane?

Reply:
Not sure if I got what you mean here, the above setting is for automatically opening both panes when opening a new tab
I'll for sure add an option to the right click to open in the right pane

Reply:
I misunderstood what this setting does 😂